pulp_ansible/app/viewsets.py

```diff
@@ -224,6 +388,43 @@ class CollectionVersionSignatureViewSet(NoArtifactContentUploadViewSet):
     filterset_class = SignatureFilter
     queryset = CollectionVersionSignature.objects.all()
     serializer_class = CollectionVersionSignatureSerializer
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "create",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_perms:ansible.add_collection",
+            },
```

Comment:
I would repeat the same access policy I linked above for pulp_file content here. CollectionVersionSignatures also don't have a update or destroy endpoint.